features/step_definitions/comment_steps.rb

```diff
@@ -302,3 +302,8 @@
     click_link("Yes, delete!") # TODO: Fix along with comment deletion.
   end
 end
+
+When /^I mark the comment as spam$/ do
```

Comment:
We're trying to shift away from regular expressions in our step definitions whenever possible -- can you use a Cucumber expression here?

```suggestion
When "I mark the comment as spam" do
```

Reply:
Sure.

For future reference, are these changes in coding standards documented somewhere?

Reply:
Yup! You can find them in the [our Cucumber Standards section of the Automated Testing wiki page](https://github.com/otwcode/otwarchive/wiki/Automated-Testing#our-cucumber-standards).